_build/partials/about.hbs

```diff
@@ -31,5 +31,8 @@
     <p>If you want to add a collection or example of how to use a collection to this registry, please follow the instructions on the <a href="https://github.com/{{githubRepo}}">Sentinel Hub Public Collections GitHub repository</a>.</p>
 
     <p>Unless specifically stated in the applicable collection's documentation, collections available through the Sentinel Hub Public Collections are not provided and maintained by Sentinel Hub. Collections are provided and maintained by a variety of third parties under a variety of licenses. Please check collection licenses and related documentation to determine if a collection may be used for your application.</p>
+
+    <h4>Access to data</h4>
+    <p>To access the data collections, please follow the example notebook on the <a href="https://github.com/tree/main/examples/how_to_access_public_collections.ipynb">Sentinel Hub Public Collections GitHub repository</a>.</p>
```

Comment:
Not found. the `sentinel-hub/public-collections` and possibly some more is missing from the url.

Reply:
It the notebook that will be added to the repository after this pull request is merged. Is there a way to have the url before the file is added to the repo?

Reply:
Removed the link for this merge. Will add the link after.